filter_lists/regional.json

```diff
@@ -65,6 +65,17 @@
         "base64_public_key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuYBXbfloR5HddFlg80U8+pf5TqfFJQAf1bL4myp9KfGggwqrjuRzIkPOD8J8IvCp8tWv2f4QK9sAPHhtV6w1cnYX24lKxrQ/lHHAV6/CEcFa+2Yk7cRLKDC10H3r4FMRoCeAy/ruTjVPfIw+GuAfFYl1qYWBNxvW7XXw7cCIIYL4j82YQF6HjsWbTT+QHLCR6h66wvIyVQC9ppjJPxDaEevjt4tohEFAB1NBC+Wxt8H/P5r5ayNcLnb9Ygt75haYL8VWZOJhO/neSTyuidTFG5ox2Ruc6TXP8t0IqpVtiZUDkx1jzUakIHoKNMBc7oz3P/SQ4AanZsIliJobXFeUiQIDAQAB",
         "desc": "Removes advertisements from Czech and Slovak websites"
     },
+    {
+        "uuid": "2BF6F16F-3214-4F55-B148-8DBB52389D1A",
+        "url": "https://raw.githubusercontent.com/tomasko126/easylistczechandslovak/master/filters_ublock.txt",
+        "title": "CZE, SVK: EasyList Czech and Slovak (Supplemental uBO list)",
```

Comment:
Do we need to specify `(Supplemental uBO list)` in the title? I think this already is true of most of the regional lists, where we leave it off.